### DISCUSSION TITLE
Option to have multiple summaries based on currency of exchange

### DISCUSSION THREAD

**Discussion by varunasthana92** [achannarasappa/ticker#192] [Ideas]
As the current version v4.2.1 supports the displays of different exchanges in their native currency (while some formatting issues are there in this mode), the overall summary is still combined to a single currency.
It would be great to have multiple summaries in the same terminal.

Current workaround: maintain different configs for different currencies and use --config flag

**Comment by achannarasappa** [comment]
Thanks for sharing this suggestion

My thinking is that #99 would address this since it would allow for multiple summaries albeit on different screens/pages. It would still be up to users to group their holdings/watchlist by exchanges for the use case described here though.
